lib/clickhousex/codec/values.ex

```diff
@@ -104,6 +104,11 @@ defmodule Clickhousex.Codec.Values do
     ["'", escape(param), "'"]
   end
 
+  defp escape(s) when is_map(s) do
+    m = Jason.encode!(s)
```

Comment:
can it be `Jason.encode_to_iodata!/1`?

Reply:
Yes! Thank you for the suggestion!

Reply:
Wouldn't JSON iodata still need to be escaped?

Reply:
Sorry for the confusion. I think the previous approach with `Jason.encode!` is the way to go since escaping iodata seems like too much work.